Core/src/org/sleuthkit/autopsy/contentviewers/ContactArtifactViewer.java

```diff
@@ -158,43 +194,81 @@ private void initComponents() {
         gridBagConstraints = new java.awt.GridBagConstraints();
         gridBagConstraints.gridx = 0;
         gridBagConstraints.gridy = 7;
+        gridBagConstraints.gridheight = 2;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.VERTICAL;
+        gridBagConstraints.weighty = 0.1;
+        add(interPanelfiller, gridBagConstraints);
+
+        personasLabel.setFont(personasLabel.getFont().deriveFont((personasLabel.getFont().getStyle() | java.awt.Font.ITALIC) | java.awt.Font.BOLD, personasLabel.getFont().getSize()+2));
+        org.openide.awt.Mnemonics.setLocalizedText(personasLabel, org.openide.util.NbBundle.getMessage(ContactArtifactViewer.class, "ContactArtifactViewer.personasLabel.text")); // NOI18N
+        personasLabel.setMaximumSize(new java.awt.Dimension(60, 19));
+        personasLabel.setMinimumSize(new java.awt.Dimension(60, 19));
+        personasLabel.setPreferredSize(new java.awt.Dimension(60, 19));
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 8;
+        gridBagConstraints.gridwidth = 3;
+        gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+        gridBagConstraints.insets = new java.awt.Insets(6, 19, 0, 0);
+        add(personasLabel, gridBagConstraints);
+
+        personasPanel.setLayout(new java.awt.GridBagLayout());
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 9;
+        gridBagConstraints.gridwidth = 4;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.VERTICAL;
+        gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+        gridBagConstraints.insets = new java.awt.Insets(6, 19, 0, 0);
+        add(personasPanel, gridBagConstraints);
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 10;
         gridBagConstraints.gridwidth = 4;
         gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
         gridBagConstraints.weighty = 1.0;
-        add(filler1, gridBagConstraints);
+        add(bottomFiller, gridBagConstraints);
         gridBagConstraints = new java.awt.GridBagConstraints();
         gridBagConstraints.gridx = 4;
         gridBagConstraints.gridy = 0;
         gridBagConstraints.gridheight = 8;
         gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
         gridBagConstraints.ipadx = 2;
         gridBagConstraints.weightx = 1.0;
-        add(filler2, gridBagConstraints);
+        add(rightFiller, gridBagConstraints);
     }// </editor-fold>//GEN-END:initComponents
 
     @Override
     public void setArtifact(BlackboardArtifact artifact) {
 
         // wipe the panel clean
         this.removeAll();
+        
+        reset();
         initComponents();
```

Comment:
initComponent should not be called in setArtifacts.  It should only be called in the constructor.